music_assistant/controllers/media/playlists.py

```diff
@@ -85,6 +93,8 @@ async def create_playlist(
         else:
             provider = self.mass.get_provider("builtin")
 
+        if provider and not provider.is_streaming_provider:
+            validate_playlist_filename(name)
```

Comment:
```suggestion
        if "/" in name or "\" in name or ".." in name:
            msg = f".name} is not a valid Playlist name"
            raise InvalidDataError(msg)
```

just always do this check - the name may only be a name and nothing else